### PR TITLE
Next Generation Documentation - BLOCKING

### PR DESCRIPTION
**Pull request by aeneasr**
# BEFORE MERGING

* [ ] Ory CLI docs generation github action must be updated to master https://github.com/ory/ci/commit/89aa7413605c9a8843f73a46afb7076b4c79a665#diff-bf7d48ba3e110b48c759118997434fdbd25f93e76bdc1af43c543f035325c20cR10

## Tasks

* [x] Create deployment in Vercel
* [x] Set up redirects to work in Vercel
* [ ] Add CI for testing
   * [x] Ensure code examples work
   * [ ] Check whether we can use text runner
   * [x] Ensure markdown format and typos are fixed with markdownlint  
   * [ ] Ensure styles are checked with Vale @vinckr

## Release

* [ ] Set up proxy from ory.sh/docs to this vercel production URL @zeropaper

### CODE REVIEW

**Comment by zeropaper** [comment]
I think that the index of https://docs-ory.vercel.app/ could be a redirection to https://docs-ory.vercel.app/docs..

That would also make the proxy setup on ory/web much easier...

@aeneasr WDYT?

**Comment by aeneasr** [comment]
Yes, totally - we have to figure out how to do this properly so it also works with deploy previews but I think it should generally be possible

File: src/hooks/index.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  }
+}))
+
+export function getSdkUrl() {
+  const [project, setProject] = useState<Project | undefined>();
+
+  useEffect(() => {
+    sdk.listProjects().then(({data: projects}) => {
+      if (projects.length > 0) {
+        setProject(projects[0]);
```
Comment (zeropaper) [inline src/hooks/index.ts:17]: What about `project[n]`? Still WIP?

It just reminds me of a similar thing (unrelated to Ory :) ) it took me a while to fix.

**Comment by vinckr** [comment]
I have a draft for the [style guide here](https://github.com/ory/docs/pull/558), will open a PR to merge it into the next-generation branch and will include the content from the README e.g. [this](https://github.com/ory/docs/tree/next-generation#guides).

So the README will link to the style guide. 
Style guide will also include docs on testing and automation, will see to add a Vale workflow to the CI.

**Pull request merged**